Bio/GenBank/Scanner.py

```diff
@@ -1653,374 +1653,5 @@ def _feed_misc_lines(self, consumer, lines):
-     gtttatccaa gaggacttcg agatcttttg ttacatatta aaaataaata taacaatcct      1260
-     gcaattttca atcactgaaa atggtatgaa tgaattcaac gatgcaacac ttccagtaga      1320
-     agaagctctt ttgaatactt acagaattga ttactattac cgtcacttat actacattcg      1380
-     ttctgcaatc agggctggct caaatgtgaa gggtttttac gcatggtcat ttttggactg      1440
-     taatgaatgg tttgcaggct ttactgttcg ttttggatta aactttgtag attagaaaga      1500
-     tggattaaaa aggtacccta agctttctgc ccaatggtac aagaactttc tcaaaagaaa      1560
-     ctagctagta ttattaaaag aactttgtag tagattacag tacatcgttt gaagttgagt      1620
-     tggtgcacct aattaaataa aagaggttac tcttaacata tttttaggcc attcgttgtg      1680
-     aagttgttag gctgttattt ctattatact atgttgtagt aataagtgca ttgttgtacc      1740
-     agaagctatg atcataacta taggttgatc cttcatgtat cagtttgatg ttgagaatac      1800
-     tttgaattaa aagtcttttt ttattttttt aaaaaaaaaa aaaaaaaaaa aaaaaaaaa       1859
-//
-"""
-
-    print("GenBank CDS Iteration")
-    print("=====================")
-
-    g = GenBankScanner()
-    for record in g.parse_cds_features(StringIO(gbk_example)):
-        print(record)
-
-    g = GenBankScanner()
-    for record in g.parse_cds_features(StringIO(gbk_example2),
-                                       tags2id=('gene', 'locus_tag', 'product')):
-        print(record)
-
-    g = GenBankScanner()
-    for record in g.parse_cds_features(StringIO(gbk_example + "\n" + gbk_example2),
-                                       tags2id=('gene', 'locus_tag', 'product')):
-        print(record)
-
-    print("")
-    print("GenBank Iteration")
-    print("=================")
-    g = GenBankScanner()
-    for record in g.parse_records(StringIO(gbk_example), do_features=False):
-        print("%s %s %s" % (record.id, record.name, record.description))
-        print(record.seq)
-
-    g = GenBankScanner()
-    for record in g.parse_records(StringIO(gbk_example), do_features=True):
-        print("%s %s %s" % (record.id, record.name, record.description))
-        print(record.seq)
-
-    g = GenBankScanner()
-    for record in g.parse_records(StringIO(gbk_example2), do_features=False):
-        print("%s %s %s" % (record.id, record.name, record.description))
-        print(record.seq)
-
-    g = GenBankScanner()
-    for record in g.parse_records(StringIO(gbk_example2), do_features=True):
-        print("%s %s %s" % (record.id, record.name, record.description))
-        print(record.seq)
-
-    print("")
-    print("EMBL CDS Iteration")
-    print("==================")
-
-    e = EmblScanner()
-    for record in e.parse_cds_features(StringIO(embl_example)):
-        print(record)
-
-    print("")
-    print("EMBL Iteration")
-    print("==============")
-    e = EmblScanner()
-    for record in e.parse_records(StringIO(embl_example), do_features=True):
-        print("%s %s %s" % (record.id, record.name, record.description))
-        print(record.seq)
+    from Bio._utils import run_doctest
+    run_doctest(verbose=0)
```

Comment:
Are there any doctests in this file? If not, there is no reason to having these three lines of code.